web/book/src/syntax/expressions-and-operators.md

```diff
@@ -15,21 +15,87 @@ filter circumference > 10 && color != "red"
 
 ## Operator precedence
 
-This table gives formal description of operator precedence. Because function
-calls have the lowest precedence, nested function calls or arguments that start
-or end with an operator require parentheses.
+This table shows operator precedence. Use parentheses `()` to prioritize
+operations and for function calls (see below.) _Question: Are parentheses
+operators? Should they be in this table?_
```

Comment:
Yes, maybe they should be first?
@aljazerzen ?

Reply:
I'd say they are not, but it doesn't hurt to include them. And yes, they should be first, yes.

Reply:
`()` is now listed at the top. Precedence is listed as "0" - I'd look for guidance here about whether to set its priority to zero, or to set it to 1 and increase all the others.